src/main/scala/com/databricks/labs/overwatch/ParamDeserializer.scala

```diff
@@ -158,7 +158,7 @@ class ParamDeserializer() extends StdDeserializer[OverwatchParams](classOf[Overw
       getOptionDouble(masterNode, "intelligentScaling.coeff").getOrElse(1.0)
     )
 
-    val workspaceFriendlyName = getOptionString(masterNode, "workspaceFriendlyName")
+    val workspace_name = getOptionString(masterNode, "workspace_name")
```

Comment:
why `workspace_name` not `workspaceName` for variables, etc.?

Reply:
for the reason I said above, I want it to be in all the tables as "workspace_name". this is the only column in OverwatcParams that winds up moving through the other tables. I can work cleaning up the pattern after the upgrade or something but it's challenging to change it here due to the way the state of the pipeline works.